### ISSUE TITLE
Abilities like Blaze don't trigger at exactly 1/3 health

### ISSUE THREAD

**Issue by patsen29** [pagefaultgames/pokerogue#300]
Normally, you need to be at 1/3 health or less to trigger Blaze, Torrent, or Overgrow. However, the way we're checking, it only triggers at 0.33 times max HP, so a Blaziken at 17/51 (33.3%) won't trigger Blaze.

Should be a simple one line fix in ability.ts:932

```
export class LowHpMoveTypePowerBoostAbAttr extends MoveTypePowerBoostAbAttr {
  constructor(boostedType: Type) {
    super(boostedType);
  }

  getCondition(): AbAttrCondition {
    return (pokemon) => pokemon.getHpRatio() <= 0.33;
  }
}
```

Maybe use <= 0.3334, since doing something like pokemon.getHpRatio() * 3 <= 1 might still trigger floating point number rounding errors.

**Comment by Madmadness65** [comment]
Fixed via #353

**Comment by patsen29** [comment]
So instead of mons between 0.3300 and 0.3333 not trigger these abilities, now mons between 0.3334 and 0.3349 will trigger?

So 100/299 is blaze now.
